library/src/app/components/bank-account-management/bank-account-connect/bank-account-connect.component.html

```diff
@@ -0,0 +1,90 @@
+<ng-container *ngIf="configService.getConfig$() | async as config">
+  <mat-card-title>
+    <app-cybrid-logo
+      fill="{{ config.theme === 'LIGHT' ? '#0270e0' : 'white' }}"
```

Comment:
Shouldn't these colours be defined in css?

Reply:
They are now css variable.